fe2-android/app/src/test/java/com/github/dedis/popstellar/ui/home/CameraPermissionFragmentTest.java

```diff
@@ -70,6 +70,11 @@ public void denyPermissionDoesNothing() {
     assertFalse("A response was received when none was expected", receiver.received());
   }
 
+  @Test
+  public void openAddWitnessSetsWitnessOnLAO(){
+
+  }
+
```

Comment:
Same issue as above. I don't like empty functions pushed on the main branch.

Reply:
Yup agreed